inject.js

```diff
@@ -19,7 +19,7 @@
         imgur.com
         teams.microsoft.com
       `.replace(regStrip,''),
-      ratechange: `
```

Comment:
I discovered `ratechange` to be ambiguous so I changed it to `blacklistrc` which means `blacklist rate change`